submissions/ORybka/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="./assets/kitty.svg" type="image/svg">
+  <link rel="stylesheet" href="style.css">
+  <title>Hooli-popup</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <nav class="navbar">
+        <ul class="menu">
+          <li class="menu-item">
+            <a href="#" class="menu-item-link">
+              Mail
+            </a>
+          </li>
+          <li class="menu-item">
+            <a href="#" class="menu-item-link">
+              Images
+            </a>
+          </li>
+          <li class="menu-item">
+            <input class="menu-label" type="checkbox" id="menu-btn">
+            <label class="menu-img" for="menu-btn">
+              <img src="./assets/menu.svg" alt="menu" />
+            </label>
+            <div class="popup">
+              <ul class="popup-menu">
+                <li>
+                  <a href="#" class="popup-item" tabindex="1">
```

Comment:
No need to use `tabindex` here. It's redundant in this case. More info [here](https://developer.mozilla.org/uk/docs/Web/HTML/%D0%97%D0%B0%D0%B3%D0%B0%D0%BB%D1%8C%D0%BD%D1%96_%D0%B0%D1%82%D1%80%D0%B8%D0%B1%D1%83%D1%82%D0%B8/tabindex)